packages/common/src/getFeeRef.ts

```diff
@@ -0,0 +1,8 @@
+import { CreateFeeRefOptions, FeeRef, createFeeRef } from "./createFeeRef";
+
+let feeRef: FeeRef | undefined;
```

Comment:
this seems to be a global cache regardless of fee args, which means one client could end up with the cached values of another client?